apps/web/src/server/applications/case/representations/representation/under-18/under-18.view-model.js

```diff
@@ -65,10 +65,13 @@ const listOfRadioOptions = [
  * @param {boolean?} rep.under18
  * @return repTypes
  */
-const getListOfOptions = ({ under18 = null }) =>
-	listOfRadioOptions.map((option) => {
-		if (option.value === under18) {
-			option.checked = true;
+export const getListOfOptions = ({ under18 }) => {
+	const under18Options = JSON.parse(JSON.stringify(listOfRadioOptions));
```

Comment:
why ?

Reply:
The objects states were being persisted. When going back and selecting a new option the previous selected checked value stayed set to true.

Reply:
Okay thats my fault for not putting the array in the func. 

3 options: 
1. Put the array in the func
2. Object.Assign the return 
3. Spread the return

Spread like this maybe: 
```
return: {
  ...option,
  option.value: option.value === under18
}
```